drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -373,6 +373,17 @@ def service_categories
       Hmis::Hud::CustomServiceCategory.all
     end
 
+    field :service_types, Types::HmisSchema::ServiceType.page_type, null: false do
+      filters_argument HmisSchema::ServiceType
+    end
+    def service_types(filters: nil)
+      raise 'Access denied' unless current_user.can_configure_data_collection?
+
+      scope = Hmis::Hud::CustomServiceType.all
+      scope = scope.apply_filters(filters) if filters
+      scope.order(:name)
```

Comment:
I'm not sure if name is unique. Maybe include id as tie breaker:
```suggestion
      scope.order(:name, :id)
```